style.css

```diff
@@ -415,6 +431,53 @@ nav {
   outline-color: #396df2;
 }
 
+.popup {
```

Comment:
Kindly stick to the design as the previous reviewer requested. One of the main goals of this project is to implement the provided design.
Please check the spacing/margin/padding around the popup on the Figma design:
<img width="428" alt="Screenshot 2022-12-22 at 9 13 58 PM" src="https://user-images.githubusercontent.com/25887253/209200324-24a88a76-10b1-463b-ae46-95ffd8583e4c.png">

This one is yours. Still doesn't match the requirements.
<img width="1440" alt="Screenshot 2022-12-22 at 9 17 46 PM" src="https://user-images.githubusercontent.com/25887253/209200939-57bda7b8-9890-4e26-b0d1-6ed476a07e1a.png">